elasticsearch/testcontainers/elasticsearch/__init__.py

```diff
@@ -12,7 +12,7 @@
 #    under the License.
 import logging
 import re
-import urllib
+import urllib.request
```

Comment:
Why specify it like this, if we use it fully qualified below?

Reply:
Yeah this works because another python module does the full import, so it's dependent on global state probably not what you want

Reply:
I posted a reproducer above/below